packages/front-end/components/Experiment/MetricsOverridesSelector.tsx

```diff
@@ -448,6 +463,128 @@ export default function MetricsOverridesSelector({
                   </div>
                 </div>
 
+                <div className="row mt-1">
+                  <div className="col">
+                    <span className="uppercase-title">Bayesian Priors</span>{" "}
+                    <span className="small text-muted">(Bayesian only)</span>
+                  </div>
+                </div>
+
+                <div className="row">
+                  <div className="col border mx-1 mt-1 mb-2 px-2 py-1 rounded">
+                    <div className="form-inline my-1">
+                      <input
+                        type="checkbox"
+                        className="form-check-input"
+                        {...form.register(
+                          `metricOverrides.${i}.properPriorOverride`
+                        )}
+                        id={`toggle-priorOverride_${i}`}
+                        disabled={!hasRegressionAdjustmentFeature}
+                      />
+                      <label
+                        className="small mr-1 cursor-pointer"
+                        htmlFor={`toggle-priorOverride_${i}`}
+                      >
+                        Override metric-level settings
+                      </label>
+                    </div>
+                    <div
+                      style={{
+                        display: form.watch(
+                          `metricOverrides.${i}.properPriorOverride`
+                        )
+                          ? "block"
+                          : "none",
+                      }}
+                    >
+                      <div className="d-flex my-2 border-bottom"></div>
+                      <div className="form-group mt-1 mb-2 mr-2 form-inline">
+                        <label
+                          className="mr-1 small"
+                          htmlFor={`toggle-properPrior_${i}`}
+                        >
+                          Use proper prior for this metric
+                        </label>
+                        <Toggle
+                          id={`toggle-properPrior_${i}`}
+                          value={
```

Comment:
TODO ensure we the value in the form matches what we show in the UI when turning on the override.